.github/ISSUE_TEMPLATE/usability.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Usability Improvement
+about: Make Bevy more fun to use!
+title: ''
+labels: C-Usability, S-Needs-Triage
+assignees: ''
+---
+
+## What inconsistency, frustration or unpleasant surprise did you encounter?
```

Comment:
I think this header can be more pleasantly (and more completely) summarized as:

```md
## How can we improve Bevy's usability?
```

Then the content can describe the various scenarios. Something like:

```
Was a feature or behavior missing, inconsistent, or frustrating in some way?
Try to provide specific examples if you can. These could be code snippets, log files,
screenshots, file attachments, etc.
```

Reply:
I think it's important to split out "what was the problem" from "what is your proposed solution". I tend to find it elicits clearer feedback, and opens the conversation to more creative and thoughtful solutions.

I'll clean up the language though :)

Reply:
Ah yeah that title was definitely combining the two categories (which wasn't my intent ... but it does read that way).